sdk/containerregistry/Microsoft.Azure.ContainerRegistry/src/Generated/AzureContainerRegistryClientExtensions.cs

```diff
@@ -69,65 +81,355 @@ public static async Task<RepositoryTags> GetTagListAsync(this IAzureContainerReg
             /// <param name='reference'>
             /// A tag or a digest, pointing to a specific image
             /// </param>
+            /// <param name='payload'>
+            /// Manifest body, can take v1 or v2 values depending on accept header
+            /// </param>
             /// <param name='cancellationToken'>
             /// The cancellation token.
             /// </param>
-            public static async Task<Manifest> GetManifestAsync(this IAzureContainerRegistryClient operations, string name, string reference, CancellationToken cancellationToken = default(CancellationToken))
+            public static async Task<object> CreateManifestAsync(this IAzureContainerRegistryClient operations, string name, string reference, Manifest payload, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
>object [](start = 37, length = 6)

The earlier method returned a Manifest which might have been more useful in the chaining. Ideally having it return a a ManifestResponse which hopefully has the server side digest would be more useful.